corehq/apps/reports/templates/reports/filters/location_async.html

```diff
@@ -8,33 +8,34 @@
     Same with the .report-filter-location-async below.
 -->
 <div class="row">
-    <div class="form-group">
-        <label class="control-label {{ css_label_class }}">{% trans "Location Filter" %}</label>
-        <div class="{{ css_field_class }}">
-            <div class="col-sm-12">
-                <select class="form-control"
-                        data-bind="value: show_location_filter">
-                    <option value="y">{% trans "On" %}</option>
-                    <option value="n">{% trans "Off" %}</option>
-                </select>
+    <div class="col-sm-12">
+        <div class="form-group">
+            <label class="control-label {{ css_label_class }}">{% trans "Location Filter" %}</label>
+            <div class="{{ css_field_class }}">
+                    <select class="form-control"
+                            data-bind="value: show_location_filter">
+                        <option value="y">{% trans "On" %}</option>
+                        <option value="n">{% trans "Off" %}</option>
+                    </select>
             </div>
         </div>
     </div>
 </div>
 {% endif %}
 
 <div class="row">
-    <div class="form-group report-filter-location-async"
-         data-bind="visible: show_location_filter_bool()"
-         data-make-optional='{{ make_optional|JSON }}'
-         data-loc-id='{{ loc_id }}'
-         data-location-url='{{ api_root }}'
-         data-hierarchy='{{ hierarchy|JSON }}'
-         data-auto-drill='{{ auto_drill|JSON }}'
-         data-locs='{{ locations|JSON }}'>
-        <label class="control-label {{ css_label_class }}">{{ control_name }}</label> {# this is wrong, see filters/single_option.html for a proper implementation #}
-        <div class="{{ css_field_class }}">
-            <div class="col-sm-12">
+    <div class="col-sm-12">
+        <div class="form-group report-filter-location-async"
+             data-bind="visible: show_location_filter_bool()"
+             data-make-optional='{{ make_optional|JSON }}'
+             data-loc-id='{{ loc_id }}'
+             data-location-url='{{ api_root }}'
+             data-hierarchy='{{ hierarchy|JSON }}'
+             data-auto-drill='{{ auto_drill|JSON }}'
+             data-locs='{{ locations|JSON }}'>
+            {# this is wrong, see filters/single_option.html for a proper implementation #}
```

Comment:
what is this wrong for?

Reply:
no clue, this comment dates back to at least 2013: https://github.com/dimagi/commcare-hq/pull/970/